src/config/cli.rs

```diff
@@ -66,12 +66,28 @@ pub struct Cli {
     /// Enable default settings for debugging contracts.
     pub debug_mode: bool,
 
+    #[arg(long, default_value = "true", default_missing_value = "true", num_args(0..=1), help_heading = "Debugging Options")]
+    /// If true, prints node config on startup.
+    pub show_node_config: Option<bool>,
```

Comment:
Why do we need this? This should always show unless silent is used?

Reply:
No, we don’t need it when using with hardhat tests and scripts. We only need console.log in that case